crates/pallet-domains/src/staking.rs

```diff
@@ -893,34 +886,34 @@ pub(crate) fn do_unlock_operator<T: Config>(operator_id: OperatorId) -> Result<(
             // if the withdrawals has share price noted, then convert them to SSC
             // if no share price, then it must be intitated in the epoch when operator was slashed,
             // so get the shares as is and include them in the total staked shares.
-            let (amount_ready_withdraw, shares_withdrew_in_current_epoch) =
-                calculate_withdraw_share_ssc::<T>(operator_id, nominator_id.clone());
+            let mut withdrawal =
+                Withdrawals::<T>::take(operator_id, nominator_id.clone()).unwrap_or_default();
```

Comment:
If there is no withdrawal for this nominator perhaps we should skip `do_convert_previous_epoch_withdrawal` and use zero for `(amount_ready_to_withdraw, shares_withdrew_in_current_epoch)`